src/guides/v2.3/install-gde/install/cli/install-cli-subcommands-maint.md

```diff
@@ -85,6 +85,42 @@ where
 
 `--none` clears the list.
 
+## For Multistore setups {#instgde-cli-multistore}
+
+When we need different layout and localized content, general speculation is that there is not an out of the box feature. But, there is one. The error processor accepts an parameter to change the error page template.
+
+```php
+if (isset($_GET['skin'])) {
+    $this->_setSkin($_GET['skin']);
+}
+```
+
+This can be passed via rewrite rule in .htaccess and needs no further changes.
+
+## For $_GET['skin'] parameter {#instgde-cli-multistore-getskin}
```

Comment:
This heading has equal weight to the previous. It looks like a completely separate section. if this is part of the previous section then lower the weight so it's clear it is part of the previous section